tests/conftest.py

```diff
@@ -80,15 +95,22 @@ def pgstac(database):
     print("Running to PgSTAC migration...")
     with PgstacDB(dsn=connection) as db:
         migrator = Migrate(db)
-        version = migrator.run_migration()
+        version = migrator.run_migration(pgstac_version)
         assert version
         print(f"PgSTAC version: {version}")
 
-        print("Load items and collection into PgSTAC")
-        loader = Loader(db=db)
-        loader.load_collections(collection)
-        loader.load_collections(collection_maxar)
-        loader.load_items(items)
+    with psycopg.connect(connection) as conn:
+        with conn.cursor() as cur:
+            for col in load_json(collection):
+                cur.execute(
+                    "SELECT * FROM pgstac.create_collection(%s::jsonb);", (col,)
+                )
+            for col in load_json(collection_maxar):
+                cur.execute(
+                    "SELECT * FROM pgstac.create_collection(%s::jsonb);", (col,)
+                )
+            for item in load_json(items):
+                cur.execute("SELECT * FROM pgstac.create_item(%s::jsonb);", (item,))
```

Comment:
we can't use pypgstac loader because there is a version check and we can't use pypgstac version 0.9.5 to load on a 0.7.1 pgstac